doc/source/examples/analysis/alignment_and_rms/rmsd.ipynb

```diff
@@ -566,10 +579,23 @@
    "source": [
     "R_charge = rms.RMSD(u, u,\n",
     "                    select='protein and name CA',\n",
-    "                    weights=ag.charges)\n",
+    "                    groupselections=[CORE, LID, NMP],\n",
+    "                    weights=ag.resids,\n",
+    "                    weights_groupselections=[core.resids, \n",
+    "                                             lid.resids, \n",
+    "                                             nmp.resids])\n",
     "R_charge.run()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can see in the below graph that the NMP domain has much higher RMSD than before, as each particle is weighted by its residue number. Conversely, the CORE domain doesn't really change much. We could potentially infer from this that residues later in the NMP domain (with a higher residue number) maintain mobility throughout the trajectory, whereas residues later in the CORE domain do not seem to contribute significantly to the RMSD.\n",
```

Comment:
"maintain mobility", I guess more of a personal preference, I guess I'd use "are mobile during the length of the trajectory"?